packages/app-desktop/gui/Sidebar/styles/sidebar-header-button.scss

```
@@ -1,29 +1,20 @@
-
-	background-color: transparent;
-	font-size: var(--joplin-toolbar-icon-size);
-	color: var(--joplin-color2);
+  padding: 4px 8px;
+  border: none;
 
-	&:hover {
-		color: var(--joplin-color-hover2);
-		background: none;
-	}
+  background-color: transparent;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)